addons/orbat/functions/fnc_createBriefingPage.sqf

```diff
@@ -261,7 +261,21 @@ _fnc_processOrbatTrackerBriefingRawData = {
                         };
                     };
 
-                    private _unitText = format["      %3<img image='%1' height='16'></img> %2 (", _unitImg, name _x, _indent];
+                    private _memberRole = "";
+                    private _memberPrep01 = roleDescription _x;
+                    if (["@",_memberPrep01] call BIS_fnc_inString) then {
```

Comment:
Regex is fun https://regex101.com/r/XDwR7v/1

Reply:
Regex frustrates me to no end.
If you want to provide the code I'll use it, but I'm happy with this as it is now, and since it runs at mission start execution time is not too relevent.

Reply:
Dammit, my curiousity got the better of mem I'll do it